packages/showcase/src/app/utility-classes/spacing/spacing.component.ts

```diff
@@ -1,11 +1,105 @@
-import { ChangeDetectionStrategy, Component } from "@angular/core";
+import {
+  ChangeDetectionStrategy,
+  Component,
+  computed,
+  CUSTOM_ELEMENTS_SCHEMA,
+  inject,
+  input
+} from "@angular/core";
+import { CommonModule } from "@angular/common";
+import { Router } from "@angular/router";
+
+import type { ChCheckboxCustomEvent } from "@genexus/chameleon-controls-library";
+
+import { spacingMetadata } from "./metadata";
+import { CSS } from "../../../common/html";
+import { CodeSnippetComponent } from "../../../user-controls/code-snippet/code-snippet.component";
 import { RuntimeBundlesComponent } from "../../../user-controls/runtime-bundles/runtime-bundles.component";
+import { CommonSnippets } from "../../../user-controls/common-snippets/common-snippets.component";
 
 @Component({
-  selector: "utility-classes-spacing",
+  selector: "components-spacing",
   templateUrl: "./spacing.component.html",
   changeDetection: ChangeDetectionStrategy.OnPush,
-  imports: [RuntimeBundlesComponent, RuntimeBundlesComponent],
-  host: { class: "main-content" }
+  imports: [
+    CommonModule,
+    CodeSnippetComponent,
+    RuntimeBundlesComponent,
+    CommonSnippets
+  ],
+  host: { class: "main-content", ngSkipHydration: "true" },
+  schemas: [CUSTOM_ELEMENTS_SCHEMA],
+  styleUrls: ["./spacing.component.scss"]
 })
-export class SpacingComponent {}
+export class SpacingComponent {
+  router = inject(Router);
+
+  hiddenGrids = input<string>("");
+
+  /**
+   * This map is useful for rendering checkboxes to determine whether a
+   * spacing section must be rendered.
+   */
+  spacings = computed(() => {
+    const newspacings = new Map<string, boolean>([
+      ["Block Inline", true],
+      ["Block", true],
+      ["Block start", true],
+      ["Block end", true],
+      ["Inline", true],
+      ["Inline start", true],
+      ["Inline end", true],
+      ["Components", true]
```

Comment:
"Components" doesn't describe what we are seeing in the section. We should improve the title.

```suggestion
      ["Components with spacing support", true]
```